etc/workflows/studio-upload.xml

```diff
@@ -202,14 +202,37 @@
       </configurations>
     </operation>
 
+    <!-- Cut the videos -->
+
+    <operation
+        id="clone"
+        exception-handler-workflow="partial-error">
+      <configurations>
+        <configuration key="source-flavor">smil/cutting</configuration>
+        <configuration key="target-flavor">smil/tmp</configuration>
+      </configurations>
+    </operation>
+
+    <operation id="editor"
+      exception-handler-workflow="partial-error"
+      description="Cut the recording according to the edit decision list">
+      <configurations>
+        <configuration key="source-flavors">*/prepared</configuration>
+        <configuration key="smil-flavors">smil/tmp</configuration>
+        <configuration key="target-smil-flavor">smil/tmp</configuration>
+        <configuration key="target-flavor-subtype">trimmed</configuration>
+        <configuration key="skip-if-not-trimmed">true</configuration>
```

Comment:
I tested it and it worked. And from my understanding of the code of the `editor`-WOH, that is expected: Before the operation is actually skipped, the media package elements should be cloned, and the clones be given the correct flavors.

However, I **was** unsure about this option, so if anyone has evidence for it not working or for my understanding to be wrong, it would be greatly appreciated. :smile: